README.md

```diff
@@ -29,7 +29,14 @@ This is a resource for anyone visiting Denver or people recently new to the Denv
+### Boulder
+Boulder is an elite college town. Maybe more accurately, it is an elite town, and a college town. The best description I have personally heard is "Wow, it's like Berkeley, without all the dirty parts." It is an interesting city because of its conflicting natures. The best aspects of Boulder are that it is a pretty chill place to live. You can bike everywhere, safely and comfortably, in some of the most beautiful surroundings in the country. Seriously. They snow-plow the bike paths before they plow the roads. There are busses that go everywhere with cute names like the Skip, Hop and Dash! There is the culture and stuff-to-do that comes from having an R1 university in the town. You are literally right next to the mountains, as in, you get to the edge of town and you are climbing. It's an almost perfect blend of nature, city and small town. The recreation centers are excellent and affordable, but you won't need them because you'll just be outside all the time.
+However! There is a strata of Boulder that is all the worst aspects of high-money liberalism you can find. People who feel their social debt is paid by donating $20,000 to the Human Society, so they can feel ok when they encourage police to round-up homeless people. Who vote for every education funding bill, but vote against more high-density housing where teachers could afford to live. Who smoke organic Yves Saint Laurent cigarettes in their hybrid SUVs on the way to yoga class after their cellular-fat-freezing session. This group is best represented by the website [Stay Out of My Namaste Space](http://stayoutofmynamastespace.com). As a result, Boulder has a very high cost of living compared to Denver. Also, Google is about to build a massive campus in town, so expect San Francisco-esque (for the region) prices soon.
+### Commuting
+Due to the nearing completion of the bus/HOV/toll lane between the two cities, the best ways to commute are by carpool or bus. If your employer can get you an EcoPass or offer pre-tax transportation benefits, that would be a good perk. The busses are big Greyhound-style coach busses with cargo compartments, so even if the bike rack is full they can put a bike into the cargo areas, making bike/bus commuting convenient. Another option is to use [B-Cycle](http://denver.bcycle.com) on either end of the bus ride. Otherwise you're going to be sitting in heavy traffic, or paying a decent chunk of money ($8 each way I think?) every day to not sit in it. The commute by bus takes under an hour except on days with heavy snow, when the bus schedule gets buried and nobody can find it so the busses just come when they come.
```

Comment:
Just another note on the use of "I" - let's try to confirm that pricing and alter that language.